spec/routing/hq/partners_routing_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe 'Hq::Partners routing', type: :routing do
+  specify { expect(get: '/hq/partners').          to be_routable }
+  specify { expect(post: '/hq/partners').         to be_routable }
+  specify { expect(get: '/hq/partners/new').      to be_routable }
+  specify { expect(get: '/hq/partners/1').        to be_routable }
+  specify { expect(get: '/hq/partners/1/edit').   to be_routable }
+  specify { expect(put: '/hq/partners/1').        to be_routable }
+  specify { expect(patch: '/hq/partners/1').      to be_routable }
+  specify { expect(delete: '/hq/partners/1'). not_to be_routable }
```

Comment:
I think its a good idea. I wouldn't miss it if it wasn't there but these little things can stop you losing your cadence when in your workflow.